aspnet/tutorials/mvc-with-entity-framework.rst

```diff
@@ -0,0 +1,394 @@
+
+To find the project directory, in Visual Studio you can right-click the project name (ContosoBooks) in the **Solution Explorer** and select **Open Folder in File Explorer**. Copy your project path from **File Explorer** so you can copy it to the **Command Prompt**. For example, enter the following from the **Command Prompt** to change directories::
+
+	cd C:\Projects\ContosoBooks\src\ContosoBooks
+	
+.. note:: Make sure that you have navigated to the *ContosoBooks* folder within the *src* folder.
+
+Run each of the following commands from the **Command Prompt**:
+
+.. code-block:: none
+
+	dnu restore
+	dnx ef migrations add Initial
+	dnx ef database update
+
+.. note:: If ``dnu restore`` is not a recognized command, you may have missed a prerequisite step (or part of a prerequisites step) at the beginning of this topic. For general EF command help, enter the following in the command window: ``dnx ef -?``. For help with the ``add`` command, enter the following in the command window: ``dnx ef migrations add -?``. And, for help with the ``update`` command, enter the following in the command window: ``dnx ef database update -?``.
+	
+The .NET Version Manager (**dnvm**) is a set of command line utilities that are used to update and configure .NET Runtime. 
+
+`DNX <http://docs.asp.net/en/latest/dnx/overview.html>`_ stands for the .NET Execution Environment. The ``ef`` command is specificed in the *project.json* file of your project. For more information about ``dnvm``, ``dnu``, and ``dnx``, see `DNX Overview <http://docs.asp.net/en/latest/dnx/overview.html>`_.
+
+The "``add Initial``" command creates a migration named "Initial" that adds code to the project, allowing EF to update the database schema. The ``update`` command creates the actual database. After you run this command, the *Migrations* folder of your project will be updated as follows:
+
+.. image:: mvc-with-entity-framework/_static/16-migrations.png
+
+Also, you will be able to view the newly created database within **SQL Server Object Explorer**. 
+
+.. image:: mvc-with-entity-framework/_static/16a-database.png
+
+Adding navigation
+-----------------
+Update the navigation for the web app. From **Solution Explorer**, open the *Views/Shared/_Layout.cshtml* file. Find the following markup:
+
+.. code-block:: aspx-cs
+
+    <li><a asp-controller="Home" asp-action="Index">Home</a></li>
+    <li><a asp-controller="Home" asp-action="About">About</a></li>
+    <li><a asp-controller="Home" asp-action="Contact">Contact</a></li>
+
+Replace the above markup with the following markup:
+
+.. code-block:: aspx-cs
+
+	<li><a asp-controller="Books" asp-action="Index">Books</a></li>
+	<li><a asp-controller="Authors" asp-action="Index">Authors</a></li>
+
+The above changes will add a link to view Books and a link to view Authors. You created each of these views when you added scaffolding to the project.
```

Comment:
Tell them that's a link TH (and use this link http://www.davepaquette.com/archive/2015/06/01/mvc-6-anchor-tag-helper.aspx 
then link to my TH docs.